### ISSUE TITLE
[Visual Requirement - Bot Webchat - Home]: Luminosity ratio of Keyboard visual focus indicator for 'upload file' and 'send' buttons is less than 3:1.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4028]
User Experience:
If the luminosity ratio of keyboard visual focus indicator is less than 3:1 for controls, then it can confuse the user having visual impairment to identify the control in low vision. 

Repro Steps:
1. Open URL: https://microsoft.github.io/BotFramework-WebChat/05.custom-components/l.disable-adaptive-cards/ 
2. Page with Bot opens, Follow the dialog on this bot, and test the Adaptive Cards at the end of the dialog
3. Check and verify that step 2 is accessible and MAS compliant.

Actual Result: 
Luminosity ratio of Keyboard visual focus indicator for 'upload file' and 'send' buttons is less than 3:1.

Expected Result:
Luminosity ratio of Keyboard visual focus indicator for 'upload file' and 'send' buttons should be equal or greater than 3:1.

MAS Reference:
WCAG1.4.11 – Non-text Contrast

Test Environment:
OS: Win 10 21H1
Build: OS Build 19043.1147
Browser: Edge Chromium Version 94.0.972.2 (Official build) dev (64-bit)
#URL: https://microsoft.github.io/BotFramework-WebChat/05.custom-components/l.disable-adaptive-cards/

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS1.4.11;#Benchmark;#Accessibility;#Win10-Edge(Chromium);#E2E-HCL-BotFrameworkWebChat-Aug21;

**Comment by compulim** [comment]
Agree this need to be fixed.

Seems dupe of #4014. If yes, please close one of them, otherwise, can you describe differences between these two bugs?

**Comment by compulim** [comment]
After discuss with our design team, we will update the style of the upload/send button to more aligned with [Fluent UI Icon Button](https://developer.microsoft.com/en-us/fluentui#/controls/web/button), particularly:

- When focus via keyboard (`:focus-visible`), will add a 1px dark border
- On mouse hover, will have a light-dim background
- When clicked (both keyboard or mouse), will have a dim background

**Comment by compulim** [comment]
Copying style from Fluent UI.

![image](https://user-images.githubusercontent.com/1622400/133177812-d7c90902-e855-417e-9ddc-3809575a0b48.png)

![image](https://user-images.githubusercontent.com/1622400/133177857-96cd7cf7-c99a-419c-b4a1-8a1950a47603.png)

![image](https://user-images.githubusercontent.com/1622400/133177897-3cb769a2-e45a-4da2-8577-b43f139d15a3.png)

Normal styles.

![image](https://user-images.githubusercontent.com/1622400/133178065-240317a0-8f15-47cf-a476-980c508ea14e.png)

Styles for `:hover` and `:active`.

![image](https://user-images.githubusercontent.com/1622400/133177972-cddc4f96-65a7-4816-82a8-45f3bab68583.png)

Styles for `:focus-visible`.

![image](https://user-images.githubusercontent.com/1622400/133178027-a771ddb9-69cc-416d-87d2-095fb477e561.png)

**Comment by compulim** [comment]
I am keeping this open but resolved.

**Comment by v-shikch** [comment]
It seems as a duplicate issue of above mentioned bug of [#4014](https://github.com/microsoft/BotFramework-WebChat/issues/4014). Hence closing this bug will further track with the linked dup bug

**Comment by compulim** [comment]
Thanks @v-shikch and @KattaBhargav. Appreciated.
